src/pre-engagement-form/state.ts

```diff
@@ -0,0 +1,94 @@
+/**
+ * Copyright (C) 2021-2023 Technology Matters
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see https://www.gnu.org/licenses/.
+ */
+
+import type { PreEngagementFormDefinition } from '.';
```

Comment:
:rabbit:

Reply:
That's a funny situation, where it's importing from a sibling file, but the file is called `index.ts`. When typing:
```
import type { PreEngagementFormDefinition } from './index';
```
Lint complains, and suggests `'.'`
```
Useless path segments for "./index", should be "."eslint[import/no-useless-path-segments]
```